lib/yform/dataset/layer.php

```diff
@@ -426,7 +426,13 @@ public function getLabel( ?string $locale ){
         if( !$locale ) $locale = \rex_clang::getCurrent()->getCode();
         $lang = \rex_var::toArray( $this->lang );
         $lang = array_column( $lang,1,0 );
-        return $lang[$locale] ?: $lang[array_key_first($lang)] ?: $this->name;
```

Comment:
Was passiert denn mit 
```
return $lang[$locale] ?? $lang[array_key_first($lang)] ?? $this->name;
```

Reply:
Es gibt eine Warning: Undefined array key "en"

Reply:
In meinem Fall funktioniert es.

Reply:
Mercie, ich gehe noch mal in mich. ?? fragt ja "nicht existent" ab und ?: auf "leer". Dein Code ich weniger tricky, dafür transparent und verständlich :-)